metadata-ingestion/source_docs/redshift.md

```diff
@@ -106,8 +106,10 @@ Note that a `.` is used to denote nested fields in the YAML recipe.
 | `include_table_lineage`     |          | `True`             | Whether table lineage should be ingested.                                                                                                                                               |
 | `table_lineage_mode`        |          | `"stl_scan_based"` | Which table lineage collector mode to use                                                                                                                                               |
 | `include_copy_lineage`      |          | `True`             | Whether lineage should be collected from copy commands                                                                                                                                  |
+| `indent`                    |          | `True`             | Whether to format queries                                                                                                                                                               |
```

Comment:
Can we call this format_sql_queries or format_sql?

Reply:
yes, I just changed the name to **format_sql_queries**